static/src/javascripts/projects/common/modules/gallery/lightbox.js

```diff
@@ -56,7 +56,7 @@ class GalleryLightbox {
             });
         };
 
-        const galleryLightboxHtml = `<div class="overlay gallery-lightbox gallery-lightbox--closed gallery-lightbox--hover">
```

Comment:
What does the removal of `gallery-lightbox--closed` do, was this just unused?

Reply:
I think it was actually by error. Not too sure what it does, but we'll leave it there as there's no tests, and this state machine is pretty complex.